apps/wizarr/config.json

```diff
@@ -6,12 +6,12 @@
   "exposable": true,
   "id": "wizarr",
   "tipi_version": 5,
-  "version": "2.2.0",
+  "version": "latest",
```

Comment:
We prefer to use fixed versions for the tag. It avoids unexpected breaking automatic updated (for example 2.2.0 to 3.4.2) Our scripts will automatically update the minor and patch versions